apcd_cms/src/apps/registrations/templates/submission_form/registration_form_body.html

```diff
@@ -506,7 +506,7 @@ <h6>
                         class="integerfield"
                         id="total_claims_value_{{ent_no}}_{{r.reg_id}}"
                         inputmode="numeric"
-                        pattern="^(?=.*[1-9])[0-9]*[.]?[0-9]{1,2}$"
+                        pattern="^(0|[1-9][0-9]*)$"
```

Comment:
The diff here is wrong, because the original pattern is lost. I think we need to do this one pattern at at time and allow 0|